podman/domain/images_manager.py

```diff
@@ -270,7 +265,6 @@ def pull(
                 config for this request. auth_config should contain the username and password
                 keys to be valid.
             platform (str) – Platform in the format os[/arch[/variant]]
-            all_tags (bool) – Pull all image tags
```

Comment:
Was this intentional?  I see this is still a keyword arg in the method signature.

Reply:
Nope. Deleted.